### DISCUSSION TITLE
how to access the database when I build to executable file?

### DISCUSSION THREAD

**Discussion by Ikhlashmulya** [NativePHP/laravel#324] [Q&A] [answered]
once i build to executable file it cannot access database, there is always error 500, but when i run native:serve it can. what is the solution?

**Comment by simonhamp** [comment, marked as answer]
This sounds like a bug and it would be helpful if you could provide a bit more detail. [Please raise an issue](https://github.com/NativePHP/laravel/issues/new?labels=bug&template=bug.yml)?
